var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -20,8 +20,13 @@ class PyMorphio(PythonPackage):
     version('2.1.2', tag='v2.1.2', submodules=True, get_full_repo=True)
     version('2.0.8', tag='v2.0.8', submodules=True, get_full_repo=True)
 
+    variant('mpi', default=False, description="Use parallel HDF5 when needed")
```

Comment:
Same as the other review… why?

Reply:
Was having issues when I brought up `py-morphio` into `py-neurodamus`, it was linking against non-mpi flavour of `hdf5` and causing conflicts. Do you see any alternative to this?

Reply:
Just requiring `depends_on('hdf5')` without any `mpi` flags. Might break a whole bunch of other stuff, though

Reply:
Indeed, that's why I added the variant.

Reply:
Then at least (for now) don't directly depend on MPI so that we can undo this later.

Reply:
I think all this `~mpi` and `+mpi` should be re-evaluated.

Reply:
Yes… to be done soon™. With the direct dependency on `mpi` omitted, it's faster to grep and adjust packages.